salt/modules/neutronng.py

```diff
@@ -0,0 +1,478 @@
+# -*- coding: utf-8 -*-
+'''
+Neutron module for interacting with OpenStack Neutron
+
+.. versionadded:: Nitrogen
```

Comment:
Oh! @akashayi Sorry I didn't catch this sooner, but this should be the `Oxygen` release instead of `Nitrogen`. Can you make a follow up PR to change that?

Reply:
Woops, looks like I missed this one when I changed the others. Follow up PR incoming. Thanks for the review/merge.

Reply:
https://github.com/saltstack/salt/pull/44841 Fix up here

Reply:
Awesome! Thanks.